apps/web/components/dialog/ReportBookingDialog.tsx

```diff
@@ -0,0 +1,186 @@
+    },
+  });
+
+  const onSubmit = (data: FormValues) => {
+    reportBooking({
+      bookingId,
+      reason: data.reason,
+      description: data.description || undefined,
+      cancelBooking: data.cancelBooking,
+      allRemainingBookings: data.allRemainingBookings,
+    });
+  };
+
+  const reasonOptions = [
+    { label: t("report_reason_spam"), value: ReportReason.SPAM },
+    { label: t("report_reason_dont_know_person"), value: ReportReason.DONT_KNOW_PERSON },
+    { label: t("report_reason_other"), value: ReportReason.OTHER },
+  ];
+
+  return (
+    <Dialog open={isOpenDialog} onOpenChange={setIsOpenDialog}>
+      <DialogContent enableOverflow>
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="flex flex-row space-x-3">
+            <div className="w-full">
+              <DialogHeader title={t("report_booking")} subtitle={t("report_booking_description")} />
+              <div className="mb-4">
+                <Label htmlFor="reason" className="text-emphasis mb-2 block text-sm font-medium">
+                  {t("reason")} <span className="text-destructive">*</span>
+                </Label>
+                <Controller
+                  name="reason"
+                  control={control}
+                  rules={{ required: t("field_required") }}
+                  render={({ field }) => (
+                    <Select
+                      {...field}
+                      options={reasonOptions}
+                      onChange={(option) => {
+                        if (option) field.onChange(option.value);
+                      }}
+                      value={reasonOptions.find((opt) => opt.value === field.value)}
+                    />
+                  )}
+                />
+                {errors.reason && <p className="text-destructive mt-1 text-sm">{errors.reason.message}</p>}
+              </div>
+
+              <div className="mb-4">
+                <Label htmlFor="description" className="text-emphasis mb-2 block text-sm font-medium">
+                  {t("description")} <span className="text-subtle font-normal">({t("optional")})</span>
+                </Label>
+                <Controller
+                  name="description"
+                  control={control}
+                  render={({ field }) => (
+                    <TextArea {...field} placeholder={t("report_booking_description_placeholder")} rows={3} />
+                  )}
+                />
+              </div>
+
+              <div className="mb-4">
+                <Controller
+                  name="cancelBooking"
+                  control={control}
+                  render={({ field }) => (
+                    <div className="flex items-center">
+                      <Checkbox
+                        id="cancelBooking"
+                        checked={field.value}
+                        className="mb-2"
+                        onCheckedChange={field.onChange}
+                      />
```

Comment:
Rather than a checkbox here, we should just leave a disclaimer that a reported booking will be cancelled (as long as it is upcoming booking). That should be the default flow, we don't need a checkbox configuration at all.